incuna-sass/components/_typography.sass

```diff
@@ -0,0 +1,216 @@
+// Based on Simple Typography by Adam Marsden and contributors. 
+// Licensed under the MIT license.
+// https://github.com/AdamMarsden/simple-typography
+
+// Font Family
+$base-font-family: 'Lato' !default
```

Comment:
maybe we should leave the base-font-family as `sans-serif` or `helvetica, arial, sans-serif` stack? since if we use the stack we know people will have at least one or the other, but with Lato we would always need to include the web font to be sure. And if devs have Lato installed it would be possible to set up the project and think that font is showing because you see it but others might not see it.